src/main/java/com/harrison/controllers/SkillController.java

```diff
@@ -27,6 +35,38 @@
 
     @RequestMapping(value = "/{name}", method = RequestMethod.GET)
     public ResponseEntity<Skill> getSkill(@PathVariable("name") String name) {
+        TransactionStatus trans = transactionManager.getTransaction(new TransactionDefinition() {
+            
+            @Override
+            public boolean isReadOnly() {
+                // TODO Auto-generated method stub
+                return false;
+            }
+            
+            @Override
+            public int getTimeout() {
+                // TODO Auto-generated method stub
+                return 0;
+            }
+            
+            @Override
+            public int getPropagationBehavior() {
+                // TODO Auto-generated method stub
+                return 0;
+            }
+            
+            @Override
+            public String getName() {
+                // TODO Auto-generated method stub
+                return null;
+            }
+            
+            @Override
+            public int getIsolationLevel() {
+                // TODO Auto-generated method stub
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=squid%3AS1135)